Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -2897,7 +2897,8 @@ public async Task<ItemImageInfo> ConvertImageToLocal(BaseItem item, ItemImageInf
                 }
                 catch (HttpException ex)
                 {
-                    if (ex.StatusCode.HasValue && ex.StatusCode.Value == HttpStatusCode.NotFound)
+                    if (ex.StatusCode.HasValue
+                        && (ex.StatusCode.Value == HttpStatusCode.NotFound || ex.StatusCode.Value == HttpStatusCode.Forbidden))
```

Comment:
There are a zillion more of error codes... why don't we log an error and move on regardless of the status code?

Reply:
🤷‍♂️ just a band aid solution for now, currently testing if it fiwes the issue on my machine